x-pack/test/functional/apps/endpoint/policy_list.ts

```diff
@@ -26,7 +27,7 @@ export default function({ getPageObjects, getService }: FtrProviderContext) {
     });
     it('shows policy count total', async () => {
       const policyTotal = await testSubjects.getVisibleText('policyTotalCount');
-      expect(policyTotal).to.equal('0 Policies');
+      expect(policyTotal).to.equal('100 Policies');
```

Comment:
so, the test was usually running too fast, and this will slow it down to wait for when it will always have data (unless some other problem happens) ?  do we know that the 100 policies is a good # that won't change?  @charlie-pichette

Reply:
The test suite loads a static dataset of 100 records.  The test originally expecting 0 records but the dataset loads quickly and the page was updating the policy count before the check for the count.  This adjusts the check for the 100 policies that are in the fixed dataset.  The before function was updated to wait until the data loads.  That should remove the flaky results by ensuring the data is loaded before any of the tests execute.

Reply:
@EricDavisX Yeah, the test was doing the assertion before the data was loaded onto to the UI. For this particular area (policy list), its mocked data and [the generator](https://github.com/elastic/kibana/blob/fe21356020ee9567e3164869bae768c1220db130/x-pack/plugins/endpoint/public/applications/endpoint/store/policy_list/fake_data.ts#L49) returns back data after a timeout of 500ms. But we can expect delays with real API calls as well. 

Re: total
The fake data generator returns back a total that is calculated based on the number of items per page - `items_per_page * 10`. So for the default view (10 items per page), 100 will always be the value. Eventually, we'll move to loading data via esArchiver, but even then, the total will be known and fixed.